benchmark/notes.md

```diff
@@ -57,7 +57,7 @@ installed with homebrew.
 ### wasmi
 
 ```
-git checkout v0.31.0
+git checkout v0.32.0-beta.7
 cargo build --profile bench
 cp target/release/wasmi_cli ~/bin
```

Comment:
With respect to my comment about unhopeful inefficiencies during runtime it seems Wasmi got installed properly by the script with all needed optimizations. One thing that never occurred to me before was how important `march=native` could be on some of the hardware systems to avoid the mentioned inefficiencies in https://github.com/yamt/toywasm/issues/8#issuecomment-1989108294.

Reply:
What I mean is Cargo's `-C target-cpu=native` option. At least according to this article (https://vfoley.xyz/rust-compilation-tip/) it can sometimes help.

Reply:
interesting. i might try it when i have a chance.
however, for those runtimes i built myself here, i didn't apply such optimizations to any of them. (unless it's the default, which i haven't investigated.)
do you think wasmi is more sensitive to this kind of optimizations than other runtimes?

Reply:
I wish I had a machine where Wasmi runs badly on so I could test it on my own. I wouldn't want Wasmi to have special optimization flags that no other runtime needs, rather I would like to understand the reason and be able to make Wasmi optimizations more robust if only I knew what was causing the issues in the first place. 🙃

Reply:
> RUSTFLAGS="-C target-cpu=native"; cargo +stable build --profile bench -p wasmi_cli

i tried this. i haven't noticed any improvements.

```
spacetanuki% ./test/run-ffmpeg.sh /usr/bin/time -l wasmi_cli.target-cpu-native -
-dir .video --                                                                  
executing File(".ffmpeg/ffmpeg.wasm")::_start() ... 
       32.14 real        32.08 user         0.04 sys
           112017408  maximum resident set size      
                   0  average shared memory size 
                   0  average unshared data size                                
                   0  average unshared stack size                               
               30677  page reclaims                                             
                   0  page faults                                               
                   0  swaps                                                     
                   0  block input operations                                    
                   0  block output operations                                   
                   0  messages sent    
                   0  messages received                                         
                   0  signals received                                          
                   0  voluntary context switches  
                 660  involuntary context switches                              
        254158714732  instructions retired                                      
        129063009419  cycles elapsed                                            
            90820608  peak memory footprint                                     
spacetanuki% 
```

Reply:
It's sad that the new Wasmi runs so poorly on your machine but I see it as my job to improve its performance on all platforms if that's possible for me. Thank you for taking all your time and feedback!